src/table/styles.js

```diff
@@ -126,15 +126,19 @@ export const VisuallyHidden = styled('span')({
 // ---------- TableTotals ----------
 
 export const TotalsCell = styled(TableCell, {
-  shouldForwardProp: (prop) => prop !== 'headerStyle' && prop !== 'isTop',
-})(({ headerStyle, isTop }) => ({
-  ...headerStyle,
-  fontWeight: 'bold',
-  position: 'sticky',
-  borderWidth: isTop ? '0px 1px 2px 0px' : '2px 1px 1px 0px',
-  top: isTop && 35,
-  bottom: !isTop && 0,
-}));
+  shouldForwardProp: (prop) => prop !== 'headerStyle' && prop !== 'isTop' && prop !== 'bodyCellStyle',
+})(({ headerStyle, bodyCellStyle, isTop }) => {
+  return {
+    ...bodyCellStyle,
+    position: 'sticky',
+    backgroundColor: headerStyle.backgroundColor,
+    fontWeight: 'bold',
+    borderWidth: isTop ? '0px 1px 2px 0px' : '2px 1px 1px 0px',
+    boxShadow: isTop ? '0px 3px 5px -5px #a6a6a6' : '3px 0px 5px -5px #a6a6a6',
+    top: isTop && 35,
```

Comment:
The idea is to replace 35 with `headerStyle.height`  but up to here headerStyle does not include height( it is auto by default) so injecting the height value to headerStyle when it has already been rendered does not seem to work logically ...